src/portals/hope/features/conversations/member/MemberSummary.tsx

```diff
@@ -96,24 +97,22 @@ export const MemberSummary: React.FC<{ memberId: string }> = ({ memberId }) => {
     variables: { memberId },
   })
 
-  if (!data) {
-    return (
-      <Loadable loading>
-        <MemberPlaceholder />
-      </Loadable>
-    )
-  }
+  const { setLocale } = useTemplateMessages()
 
-  const { member } = data
+  useEffect(() => {
+    setLocale((data?.member?.pickedLocale as PickedLocale) || PickedLocale.SvSe)
```

Comment:
Probably not an issue, but this would make it cast `undefined` to `PickedLocale` - this should be more correct
```suggestion
    setLocale((data?.member?.pickedLocale || PickedLocale.SvSe) as PickedLocale)
```